gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -70,7 +70,8 @@
 ]
 # If the # of files matching the input file_pattern exceeds this value, then
 # headers will be merged in beam.
-_USE_HEADER_MERGER_THRESHOLD = 100
+_USE_BEAM_READ_HEADER_THRESHOLD = 100
+_USE_BEAM_READ_ALL_HEADER_THRESHOLD = 50000
```

Comment:
Consider renaming this to a more general _USE_BEAM_READ_ALL_THRESHOLD (and add a comment about what this does). This can be used for both VCFs and headers.

Reply:
Done.